pkg/reconciler/kubernetes/tektoninstallerset/tektoninstallerset.go

```diff
@@ -166,6 +166,12 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, installerSet *v1alpha1.T
 	// Update Ready status of Controller
 	installerSet.Status.MarkControllerReady()
 
+	// job
+	err = installer.IsJobCompleted()
```

Comment:
Could you add a log here (info nor error) as the result of this check is not going to show up in any of the existing InstallerSet status conditions.